django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -821,28 +821,21 @@ input[type="submit"], button {
     .submit-row {
         padding: 10px 10px 5px;
         margin: 0 0 15px;
-        display: flex;
         flex-direction: column;
     }
 
-    .submit-row > * {
-        width: 100%;
-    }
-
     .submit-row input, .submit-row input.default, .submit-row a {
-        float: none;
         margin: 0 0 10px;
         text-align: center;
     }
 
     .submit-row a.closelink {
-        float: none;
         padding: 10px 0;
         text-align: center;
     }
 
-    .submit-row p.deletelink-box {
-        order: 4;
+    .submit-row a.deletelink {
+        margin: 0 0 10px;
```

Comment:
`Delete` is always the last button, IMO `10px` are unnecessary because `submit-row` has already the bottom padding set to `5px`:
```suggestion
        margin: 0 0 5px;
```
so:

![image](https://user-images.githubusercontent.com/2865885/185325232-8515f127-0bad-4ae4-86be-e5da719a5d3d.png)

instead of:

![image](https://user-images.githubusercontent.com/2865885/185324797-a16ceb02-d383-42eb-ba03-2a809ee28c98.png)